frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -104,6 +110,22 @@
               @click='openEmoticon'
             )
               i.icon-smile-beam
+          tooltip(
+            v-if='ephemeral.showButtons'
+            direction='top'
+            :text='L("Attach file")'
+          )
+            button.is-icon.c-file-attachment-btn(
+              :aria-label='L("Attach file")'
+              @click='openFileAttach'
+            )
+              i.icon-paper-clip
+              input(
+                ref='fileAttachmentInputEl'
+                type='file'
+                accept='image/*,.pdf'
```

Comment:
Is this string here what determines what file types we can and cannot upload?

If so, a few requests:

1. Could you bind it to a computed property?
2. Could you make sure users can upload commonly uploaded file types? (e.g. videos, archives (.tar.gz, .tgz, .zip, .bzip2, .7z, etc.), images, pdfs)
3. If a user tries to upload an unsupported file type, display a `gi.ui/prompt` or an `alert` that shows the supported filetypes as a comma-separated list of file extensions

Reply:
@taoeffect 
I took some time for researching on this topic but was not able to find one that seems to be frequently referenced by the dev community (Or, if you know any other useful reference, let me know) so decided to fall back to [this MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types).


Further down there is a table that lists a common file type and the extension. Do you see any file extension that you think must not be supported by our group-chat attachment for a security reason?
I can put together the rest and use the list as the supported types in our app.

Reply:
@SebinSong Hmm. I do not see anything on there that we can't support, except maybe `.php`. However, as far as I know we will be saving these files under the content hash, not with their file extension, so even that should be fine. /cc @corrideat

Reply:
@taoeffect 
This is the warning modal that the app will show when an unsupported file is attached. pls let me know if you want to tweak the message content.

<img src='https://github.com/okTurtles/group-income/assets/17641213/90412c5f-47a7-4765-9dfb-d548c6c6caff' width='460'>